lib/active_merchant/billing/network_tokenization_credit_card.rb

```diff
@@ -34,6 +34,18 @@ def credit_card?
       def type
         "network_tokenization"
       end
+
+      def self.test_credit_card
```

Comment:
nit: class methods at the top of the definition (above instance methods) please.